lib/services/lists.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const _isEqual = require('lodash/isEqual');
+
+let db = require('./db');
+
+/**
+ * Appends an entry to a list and returns the updated list.
+ * @param {Array<Object>} list
+ * @param {Array<Object>} data
+ * @returns {Array<Object>}
+ */
+function addToList(list, data) {
+  return list.concat(data);
+}
+
+/**
+ * Removes an entry from a list and returns the updated list.
+ * @param {Array<Object>} list
+ * @param {Array<Object>} data
+ * @returns {Array<Object>}
+ */
+function removeFromList(list, data) {
+  const startLength = list.length;
+
+  for (const deletion of data) {
+    list = list.filter(entry => !_isEqual(entry, deletion));
+  }
+
+  if (list.length === startLength) {
+    throw new Error('Nothing was removed from the list.');
```

Comment:
Personally I feel throwing an error in this case will prove more trouble than it's worth.  Like if two editors end up removing the same item from the list that should work just fine rather than cause a 500.

not a strong opinion though

Reply:
I would tend to agree here. If the goal is to have it removed, it was already achieved.